public/admin/js/mapfiles.js

```diff
@@ -19,14 +19,13 @@ $(document).ready(function() {
 		$(activeLinkContainer).append(loadingGif);
 		
 		var params = {
-			action: 'refresh',
 			target: $(this).attr('data-target'),
 			project: $('input#project').val(),
 			mapset: $(this).attr('data-mapset')
 		}
 		
 		$.ajax({

```

Comment:
The response in case of errors has changed (for example `response.error.type` does not exists anymore). Furthermore with http status code different from 200 the error callback is called instead of success.